play/src/pages/GameInProgress.tsx

```diff
@@ -185,51 +185,16 @@ export default function GameInProgress({
   });
 
   // contents is the quill pad contents
-  const handleSubmitAnswer = async (packagedAnswer: LocalAnswer) => {
```

Comment:
Definitely starting to see an impact of the classes here

Reply:
Hopefully a god impact